src/components/dropdown/types.d.ts

```diff
@@ -0,0 +1,25 @@
+import { MouseEventHandler, ReactNode } from 'react';
```

Comment:
Could you explain the reason of using declaration file?

Reply:
Seems like it was added accidentally while creating the file.
Renamed.